README.md

```diff
@@ -156,6 +156,14 @@ Other places you might want to check out are [r/rust_gamedev][rg] and the [#rust
 
 **Note:** Any interaction with the Amethyst project is subject to our [Code of Conduct](https://github.com/amethyst/amethyst/blob/master/CODE_OF_CONDUCT.md).
 
+[![](https://sourcerer.io/fame/jojolepro/amethyst/amethyst/images/0)](https://sourcerer.io/fame/jojolepro/amethyst/amethyst/links/0)
```

Comment:
is there a reason its under your username?

Reply:
Because it needed an account to use for the api calls, so i used mine. Is this an issue?

Reply:
No I dont think so just wanted to make sure that we aren't putting something on the readme that ties to only one person.